docs/cn/Configuring-Alluxio-with-Swift.md

```diff
@@ -74,9 +72,10 @@ SoftLayer需要将`<swift-auth-model>`设置为`swiftauth`。
 
 {% include Common-Commands/stop-alluxio.md %}
 
-## 对IBM SoftLayer进行功能测试
+# 运行功能测试
```

Comment:
I see. I think English doc is correct, as it may display properly in the table of contents. Could you also update the English part?